README.md

```diff
@@ -1,8 +1,17 @@
-# Daily Real-Time Voice Inference React Native SDK
+<h1><div align="center">
+ <img alt="pipecat client react native" width="500px" height="auto" src="https://raw.githubusercontent.com/pipecat-ai/pipecat-client-react-native-daily-transport/main/pipecat-react-native.png">
+</div></h1>
 
-[RTVI-AI](https://github.com/rtvi-ai/) is an open standard for Real-Time Voice [and Video] Inference.
+[![Docs](https://img.shields.io/badge/documentation-blue)](https://docs.pipecat.ai/client/introduction)
+![NPM Version](https://img.shields.io/npm/v/@pipecat-ai/react-native-daily-transport)
 
-This library exports a VoiceClient that has the [Daily](https://www.daily.co/) transport associated.
+The official React Native client SDK for [Pipecat](https://github.com/pipecat-ai/pipecat), an open source Python framework for building voice and multimodal AI applications.
```

Comment:
The only thing I’m not sure about is whether we could improve this or not. Currently, we don’t have an SDK specifically for React Native; instead, we use the same plain JavaScript SDK, @pipecat-client/client-js.

The difference in this case lies in the Daily transport. We have one Daily transport for the web and this one for React Native.

Reply:
What do you think @markbackman ?

Reply:
Ahh, yes. Maybe we shy away from that wording then. Good point.

Reply:
I just made a change.